src/zrc/conf/base.py

```diff
@@ -56,7 +56,8 @@
     'corsheaders',
     'vng_api_common',  # before drf_yasg to override the management command
     'vng_api_common.notifications',
-    'zds_schema',  # TODO: remove once all environents have migrated
+    'vng_api_common.notifications.publish',
+    # 'zds_schema',  # TODO: remove once all environents have migrated
```

Comment:
this cannot be removed yet, we haven't done an update of stable yet I think

Reply:
uncommented zds_schema